pages/style-guide/prose.md

```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
+
+### Article Header Metadata
+
+Each article should have the following header (see below as some metatags are optional):
+
+```
+---
+title:  <article title>
+level:  [beginner|intermediate|advance]
+source:  <url of source of the material derived>
+attribution:
+  - Ralph Whitbeck <ralph@email.com>
+  - John Paul <john@email.com>
+---
+```
+
+### Code Blocks
+
+Code blocks should be set off with triple backticks and should **not** be indented. That is to say, the site prefers the use of "fenced blocks" notation supported by [gfm](http://github.github.com/github-flavored-markdown/).
```

Comment:
`gfm` does not seem like a good abbreviation for this. People might not know where we're talking about.